ceph-salt-formula/salt/ceph-salt/software.sls

```diff
@@ -8,7 +8,13 @@ install iputils:
       - iputils
     - failhard: True
 
-{% if pillar['ceph-salt'].get('upgrades', {'enabled': False})['enabled'] %}
+install lsof:
+  pkg.installed:
+    - pkgs:
+      - lsof
```

Comment:
we can move `lsof` installation to the above `pkg.installed` state

Reply:
Done